promising/promising_function.py

```
@@ -15,31 +15,30 @@ def __init__(
         self.original = func_or_class
-
-        # TODO Is maintaining these attributes here like this directly a good
-        #  idea ?
         self.start_soon = start_soon
-        self.make_parent_wait = make_parent_wait
-        self.config_inheritable = config_inheritable
+        self.children_start_soon = children_start_soon
 
     def __call__(
         self,
         *args: Any,
```

Ruff (0.15.0)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `*args`

(ANN401)